server/tests-py/queries/graphql_mutation/update/permissions/user_cannot_unpublish.yaml

```diff
@@ -0,0 +1,26 @@
+description: Try to change the published flag on a published article
```

Comment:
This test doesn't seem to be testing the `check` constraint defined above. We should testing for both a success and an error.

Reply:
Oops, thanks. I forgot to finish this and got confused because the tests were passing and I had uncommitted work locally.